src/types/media-session.ts

```diff
@@ -293,7 +294,10 @@ export class GenesysCloudMediaSession extends MediaSession {
   }
 
   _handleDataChannelMessage (event: MessageEvent) {
-    this._log('debug', 'data channel message received', event);
+    if (event.data !== this.lastDataChannelMessage?.data) {
```

Comment:
this won't actually catch anything will it? This is reference equality check which will always be falsy.

Reply:
> this won't actually catch anything will it? This is reference equality check which will always be falsy.

Actually this is fine because `this.lastDataChannelMessage` is just a reference to `event`.